### DISCUSSION TITLE
Disable theme but enable custom theme

### DISCUSSION THREAD

**Discussion by oSumAtrIX** [darkreader/darkreader#11476] [Q&A]
Maybe related: https://github.com/darkreader/darkreader/discussions/7069

## About

I am using Firefox on my Android device and use the Dark Reader extension for it. 

I want to theme a website but disable the theme that Dark Reader provides for that site by default. 

## What I tried

I have been able to add my own CSS rules, but not been able to remove the ones Dark Reader automatically applies.

## Alternative solutions

Stylus could be used as it is intended for my purpose. Unfortunately, Firefox on Android does not have this extension.

## TLDR

- Disable Dark Reader's own CSS rules
- Add my own CSS rules

**Comment by tpdi** [comment]
I'm trying to do this too. 

If you scroll to the top of the dynamic configuration file,  you'll see a "*". I'm assuming this applies either to all sites, or to any that don't have a better match in the dynamic file.

I changed this to "*.a" to see if I could turn it off, but that didn't work.

If you examine a page with dark reader on, you'll see it injects dozens of css with a prefix --dark-reader. It appears these classes are added to individual elements. 

The best (not very good) workaround I've found is to add css to override the dark reader changes I don't want.

**Reply by Hets1ng** [reply]
How did you implement custom CSS？
I use the developer tool -> Per Site Editor to change css for some websites. But it not works after i restart browser. I must click apply angin.
